collections/hub/golden_path/feature_tests/block_transfer.json

```diff
@@ -367,7 +367,7 @@
                 "requestVariables.quoteId = '{$function.generic.generateUUID}'",
                 "requestVariables.transactionId = '{$function.generic.generateUUID}'",
                 "",
-                "requestVariables.amount = -Number(environment.payerSettlementAccountBalance) - Number(environment.payerPositionBeforeTransfer) + 1000;",
+                "requestVariables.amount = -Number(environment.payerSettlementAccountBalance).toFixed(2) - Number(environment.payerPositionBeforeTransfer).toFixed(2) + 1000;",
```

Comment:
the "2" here if it is the digits after the period, using two here maybe fine for now, but we'll need to get the scale specific to each currency based on the ISO 4217 standards for this to be generic, isn't it?